exposed-java-time/src/test/kotlin/org/jetbrains/exposed/DefaultsTest.kt

```diff
@@ -400,7 +401,7 @@ class DefaultsTest : DatabaseTestsBase() {
         java.util.TimeZone.setDefault(java.util.TimeZone.getTimeZone(ZoneOffset.UTC))
         assertEquals("UTC", ZoneId.systemDefault().id)
 
-        val nowWithTimeZone = OffsetDateTime.now()
+        val nowWithTimeZone = OffsetDateTime.now().truncatedTo(ChronoUnit.MICROS)
```

Comment:
Was this change accidentally left in? If on purpose, what's the relation to the current issue?

Reply:
@bog-walk Right, sorry, I ran "gradlew test" and happened to fail there for 1 micro second difference.
So I thought it was due to rounding problem and put this code.
But it's irrelevant to this problem as you said. removed it.